xmtp/src/client.rs

```diff
@@ -22,7 +22,7 @@ use crate::{
 use std::collections::HashMap;
 use xmtp_proto::xmtp::message_api::v1::Envelope;
 
-const INSTALLATION_REFRESH_INTERVAL_NS: i64 = 10 * 60 * 1_000_000_000; // 10 minutes
+const INSTALLATION_REFRESH_INTERVAL_NS: i64 = 0;
```

Comment:
This is a hack in order to make test case (1) pass in a timely way. I have a proposal for a clean/better way of handling installation syncing that I will put up in a bit.